payments/credit-account/after-payment.md

```diff
@@ -113,11 +235,412 @@ Content-Type: application/json
+| └─➔&nbsp;`isOperational`  | `boolean` | `true`  if the transaction is operational; otherwise  `false` .                                                                                                                                              |
+| └─➔&nbsp;`operations`     | `array`   | The array of [operations][operations] that are possible to perform on the transaction in its current state.                                                                                                  |
+
+### Capture Sequence
+
+`Capture` can only be done on a authorized transaction. It is possible to do a
+part-capture where you only capture a part of the authorization amount. You can
+later do more captures on the same payment up to the total authorization amount.
+
+```mermaid
+sequenceDiagram
+  activate Merchant
+  Merchant->>-SwedbankPay: POST [CreditAccount captures]
+  activate SwedbankPay
+  SwedbankPay-->>-Merchant: transaction resource
+```
+
+## Cancellations
+
+`Cancel` can only be done on a authorized transaction. If you do cancel after
+doing a part-capture you will cancel the difference between the capture amount
+and the authorization amount.
+
+### Create cancellation transaction
+
+Perform the `create-cancel` operation to cancel a previously created - and not
+yet captured - payment.
+
+{:.code-header}
+**Request**
+
+```http
+POST /psp/creditaccount/payments/{{ page.paymentId }}/cancellations HTTP/1.1
+Host: {{ page.apiHost }}
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "transaction": {
+        "description": "Test Cancellation",
+        "payeeReference": "ABC123"
+    }
+}
+```
+
+{:.table .table-striped}
+| Required | Property                 | Type          | Description                                                                                              |
+| :------: | :----------------------- | :------------ | :------------------------------------------------------------------------------------------------------- |
+|  ✔︎︎︎︎︎  | `transaction`            | `object`      | The `object` representation of the generic [transaction resource][transaction-resource].                 |
+|  ✔︎︎︎︎︎  | └➔&nbsp;`description`    | `string`      | A textual description of the reason for the `cancellation`.                                              |
+|  ✔︎︎︎︎︎  | └➔&nbsp;`payeeReference` | `string(30*)` | A unique reference for the `cancellation` transaction. See [payeeReference][payee-reference] for details. |
+
+The `cancel` resource contains information about a cancellation transaction
+made against a payment.
+
+{:.code-header}
+**Response**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "payment": "/psp/creditaccount/payments/{{ page.paymentId }}",
+    "cancellation": {
```

Comment:
Don't we have an include that can replace all transaction code examples?

Reply:
> The include was not used because I understood that it wasnt possible to modify tables in the includes. The original include did not fit this page as some operations are not included in Credit Account.

But can't we modify the include so it suits the needs of Credit Account as well?